testnet/stacks-node/src/tests/neon_integrations.rs

```diff
@@ -0,0 +1,212 @@
+use super::{make_stacks_transfer_mblock_only, SK_1, ADDR_4, to_addr};
```

Comment:
Can you add some tests to verify that we can't exceed the epoch block compute or size budgets when mining?

Reply:
Yep -- added a test in 0dfa4cc1f